### PR TITLE
feat: Set both caller and self as controller by default

### PR DESCRIPTION
**Pull request by adamspofford-dfinity**
Jira:
> Currently, when the Cycles Wallet creates a new canister, it lists itself as the controller of that canister. Now that multiple controllers are supported, it should make the dfx identity a controller by default.

### CODE REVIEW

File: wallet/src/lib.rs [hunk 1]
```diff
@@ -443,10 +443,15 @@ mod wallet {
         struct In {
             settings: Option<CanisterSettings>,
         }
+        let mut settings = normalize_canister_settings(args.settings)?;
```
Comment (ericswanson-dfinity) [inline wallet/src/lib.rs:446]: Please take a look at create_wallet(), which calls this method settings specifically modified in order to have no controllers set.

Comment (ericswanson-dfinity) [inline wallet/src/lib.rs:446]: Please take a look at create_wallet(), which calls this method settings specifically modified in order to have no controllers set.

**Review by ericswanson-dfinity — CHANGES_REQUESTED** [review]
Would you please add a little information (in the PR description) about context?

For example, will we also be modifying dfx to allow configuring it such that all newly canisters will have as controllers the wallet, the dfx identity, and a configured II identity?  If so, this code path won't be taken in practice, so it would be useful to know in which cases it would be taken.

**Comment by adamspofford-dfinity** [comment]
I have no idea what the changes will be on the DFX end, Kyle said to just implement it on the wallet level for now.

**Review by ericswanson-dfinity — CHANGES_REQUESTED** [review]
Let's start by updating the e2e tests and go from there

**Review by ericswanson-dfinity — APPROVED** [review]

**Pull request merged**